microcosm_flask/conventions/crud.py

```diff
@@ -47,7 +47,7 @@ def configure_search(self, ns, definition):
         def search(**path_data):
             request_data = load_query_string_data(definition.request_schema)
             page = self.page_cls.from_query_string(request_data)
-            return_value = definition.func(**merge_data(path_data, request_data))
+            return_value = definition.func(**merge_data(path_data, page.to_dict()))
```

Comment:
Before my change, `crud` would just pass `load_query_string_data(definition.request_schema)` to the definition function by bypassing `Page` altogether. This is effectively equivalent to `Page.from_query_string(PageSchema()).load(request.args).data)`.

Previously we moved the offset and limit defaulting to `self.page_cls` (which is a `Page` instance) but never tied the two together so the defaulting of `offset` and `limit` went away.